### ISSUE TITLE
css: <li> color contrast doesn't meet WCAG 2.0 standards (#808080 is too light)

### ISSUE THREAD

**Issue by weedySeaDragon** [scala/docs.scala-lang#569]
color #808080 is too light for <li> elements in the body text.  It doesn't meet the WCAG 2.0 AA standards for enough contrast.  (I think using it in other places -- like the numbers for code is OK since those are ancillary to the information being presented.)  (I find the WebAIM tools for quickly [checking color contrast](http://webaim.org/resources/contrastchecker/) and for [scanning a page for issues](http://wave.webaim.org/)  to be really easy to use to check basic accessibility items.

I didn't want to do a pull request because I didn't want to do something that might unbalance the design as a whole.  But it seems on the surface as though just darkening it a bit would be fine.
Line 92 in bootstrap.css could be changed to use #606060, which would pass WCAG 2.0 AA at that size.

**Comment by SethTisue** [comment]
related: https://github.com/scala/scala-lang/pull/467

**Comment by SethTisue** [comment]
corresponding issue on scala-lang.org repo: https://github.com/scala/scala-lang/issues/509

**Comment by wenyangfu** [comment]
Since https://github.com/scala/scala-lang/issues/509 has been closed, can this also be closed?

**Comment by weedySeaDragon** [comment]
The `<li>` elements in the body text _do_ have enough contrast now.  (Yay!)

But there are still issues with the contrast.   Specifically, there's still not enough contrast with the `<a>` elements.  It's easy to check with the WAVE tool -- just put in the URL of any page and it checks and shows you issues.  http://wave.webaim.org/

### The red flags show where there are contrast issues on the _Getting Started_ page:


<img width="897" alt="screen shot 2017-11-30 at 10 56 22 am" src="https://user-images.githubusercontent.com/673794/33449740-8899957a-d5be-11e7-9993-312f932916e2.png">


---

@SethTisue Should we close this issue, and then I'll create a new issue?
The new issue(s) can be either:
(1) One issue titled "Meet minimum WCAG 2.0 usability standards," which will list all of the issues found?  (not only the contrast issue, but also the missing text for images, etc.)
or
(2) separate issues for each main area that needs to be addressed?  (one for the contrast, another issue for  missing text for images, etc.) ?

I'm open to suggestions as to whatever is the best path.

**Comment by SethTisue** [comment]
@weedySeaDragon yes, please make new issue(s), and include the information about WAVE. I don't think we want, say, 10 new tickets, other than that I don't really have an opinion about one-vs-multiple
